src/components/AdditionalSpecies.jsx

```diff
@@ -0,0 +1,188 @@
+import React, { useState, useEffect } from 'react';
+import { nanoid } from 'nanoid';
+import {
+  Heading,
+  VStack,
+  SimpleGrid,
+  GridItem,
+  Button,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  Input,
+  NumberInput,
+  NumberInputField,
+  Checkbox,
+  FormLabel,
+  FormControl,
+  IconButton,
+  Container,
+} from '@chakra-ui/react';
+import { DeleteIcon, ChevronDownIcon } from '@chakra-ui/icons';
+import DropdownSearch from './DropdownSearch';
+
+// const rows = [
+//   {
+//     id: nanoid(),
+//     check: null,
+//     name: <Input size="lg" backgroundColor="#EDF2F7" defaultValue="Beach Cast" isReadOnly />,
+//     total: (
+//       <NumberInput size="lg" defaultValue={0}>
+//         <NumberInputField />
+//       </NumberInput>
+//     ),
+//   },
+// ];
+
+const rows = new Map([[nanoid(), { name: 'Beach Cast', total: 0, isChecked: false }]]);
+
+const sp = [
+  {
+    value: 'Corvid: Common Raven',
+    label: 'Corvid: Common Raven (CORA)',
+  },
+  {
+    value: 'Cormorant: Double Crested',
+    label: 'Cormorant: Double Crested (BRAC)',
+  },
+  {
+    value: "Hummingbird: Alan's",
+    label: "Hummingbird: Alan's (ALHU)",
+  },
+  {
+    value: 'Pelican: American White',
+    label: 'Pelican: American White (AWPE)',
+  },
+  {
+    value: 'Sandpiper: Long-billed Curlew',
+    label: 'Sandpiper: Long-billed Curlew (LBCU)',
+  },
+];
+
+const AdditionalSpecies = () => {
+  const [species, setSpecies] = useState(rows);
+  useEffect(() => {
+    // Update the document title using the browser API
+    console.log(species);
+  }, [species]);
+
+  // const handleAddInputChange = event => {
+  //   event.preventDefault();
+
+  //   const fieldName = event.target.getAttribute('check');
+  //   const fieldValue = event.target.value;
+
+  //   const newTotalData = addTotalData.concat(fieldValue);
+
+  //   setSpecies(newTotalData);
+  // };
+
+  const handleCheckedItems = id => {
+    const newCheckedData = new Map(species);
+    // const currentValue = newCheckedData.get(id);
+
+    // newCheckedData.set(id, { ...currentValue, isChecked: true });
+    console.log(id);
+    setSpecies(newCheckedData);
+  };
+
+  const handleAddRow = () => {
+    const newData = {
+      name: 'Input Name',
+      total: 0,
+      isChecked: false,
+    };
+
+    const newSpecies = new Map(species);
+    newSpecies.set(nanoid(), newData);
```

Comment:
Using nanoid could result in duplicate IDs. You can either check whether the ID has already been used before using it, or assign the ID based on a counter.